client/src/components/Home.js

```diff
@@ -1,57 +1,107 @@
 import React from "react";
 import "../styles/Home.css";
-import { Header, Button } from "semantic-ui-react";
+import { Header, Button, Card, Icon } from "semantic-ui-react";
+import { Link } from "react-router-dom";
 
 const Home = props => {
   return (
     <div>
+      <div className="mainImage">
+        <img
+          src="https://trello-attachments.s3.amazonaws.com/5d571bc33416048d79622ed3/5d86265957abda1d0d4f6e3b/ae9c599fe9420c053e5195a872c25eed/hero-broca.jpg"
```

Comment:
Instead of using the url from trello, you can put the image directly into the public folder and use `./hero-broca.jpg` in the code.